vsintegration/src/FSharp.Editor/CodeFixes/AddOpenCodeFixProvider.fs

```diff
@@ -55,21 +55,26 @@ type internal AddOpenCodeFixProvider [<ImportingConstructor>] (assemblyContentPr
 
                 // attribute, shouldn't be here
                 | line when line.StartsWith "[<" && line.EndsWith ">]" ->
```

Comment:
Are the lines trimmed? If not, probably shoul trim it.

Reply:
It is trimmed at the top of this match:

`match sourceText.Lines[ insertionLineNumber ].ToString().Trim()`